source/content/integrated-composer.md

```diff
@@ -136,7 +136,66 @@ Some packages are not compatible with Composer 2. If you encounter a build error
 
 <Partial file="composer-support-scope.md" />
 
-## Troubleshooting / FAQ
+## Troubleshooting Code Syncs and Upstream Updates
+
+If you encounter an error during a code sync or if the site is missing files that should be added by Integrated Composer, the Build Log may contain information that can help you troubleshoot:
+
+1. Navigate to **<span class="glyphicons glyphicons-embed-close"></span> Code** in the **<span class="glyphicons glyphicons-wrench"></span> Dev** tab of your Site Dashboard.
+
+1. In the **Commit Log** section, find the most recent commit and click **View Log** to view the Composer command that was run and the output that was given by that command.
+
+### Dashboard Workflow Shows an Error During Sync Code or Deploying to a New Environment
+
+If there is an error in the output, it may be due to an error in the site's `composer.json` or `composer.lock` file, or there may be an issue with a Composer library the site uses.
+
+To resolve, examine the error in the log. It may be a syntax or parse error of the JSON files, or some sort of error loading a library via Composer. You can also try running the same command on your local Git checkout of the site's code and see if you can update the `composer.json` and `composer.lock` files to run the command successfully.
+
+### Upstream Updates Cannot Be Applied
+
+When you click **Apply Updates**, the process completes with the error, `Something went wrong when applying updates. View log`:
```

Comment:
The message is a red box that says, "Something went wrong when applying updates. View log". "View log is a link and the result of which is the conflict message. So I don't feel this is communicated well?

Maybe something like this?

```suggestion
When you click **Apply Updates**, the process completes with the error, `Something went wrong when applying updates. View log` Clicking on "View Log" shows this message:
```